src/haddock/modules/analysis/caprieval/__init__.py

```diff
@@ -208,15 +208,17 @@ def irmsd(self, cutoff=5.):
         ref_interface_resdic = identify_interface(self.reference, cutoff)
 
         # Load interface coordinates
-        Q, _ = load_coords(self.reference,
-                           filter_resdic=ref_interface_resdic,
-                           atoms=self.atoms,
-                           ignore_missing=self.ignore_missing)
+        _Q, _ = load_coords(self.reference,
+                            filter_resdic=ref_interface_resdic,
+                            atoms=self.atoms,
+                            ignore_missing=self.ignore_missing)
         # Move to centroids
-        Q = Q - centroid(Q)
+        _Q -= centroid(_Q)
 
         for model in self.model_list:
 
+            Q = copy.deepcopy(_Q)
```

Comment:
I think there's no need for this `deepcopy`.

Reply:
better safe than sorry?

Reply:
Added a comment for prosperity, if we ever need to debug this again it will be likely that someone (myself) will forget about this copy.